standards/code-style.adoc

```diff
@@ -338,12 +342,68 @@ if (condition) {
 }
 ----
 
-This is __bad__:
-
 [source, kotlin]
 ----
 if (condition)
-    body()  // bad!
+    body()  // WRONG!
+----
+
+Braces are not required for `when` branches and `if` statement bodies if they meet BOTH
+the following conditions:
+
+. the conditional and the branch each contain at most a single predicate
+. it fits on a single line
+
+For `if` statements, there is an additional condition that there be no `else if` or `else`
+branches.
+
+[source, kotlin]
+----
+// Okay
+if (string.isEmpty()) return
+
+// Okay
+when (value) {
+    0 -> doSpecificThing()
+    1 -> doAnotherSpecificThing()
+	else -> doGenericThing()
+}
+
+// WRONG
+if (string.isEmpty()) doEmptyThing() else doFullThing()
```

Comment:
TBH I don't actually hate this? I think with context highlighting in a modern IDE (read: Android Studio) this is still fairly clear, assuming it's still 1 line and concise. 

But I will cave to anyone with a strong opinion.